library/debug.c

```diff
@@ -410,4 +410,43 @@ void mbedtls_debug_printf_ecdh( const mbedtls_ssl_context *ssl, int level,
 }
 #endif /* MBEDTLS_ECDH_C */
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+void mbedtls_debug_print_ticket_flags( const mbedtls_ssl_context *ssl, int level,
+                      const char *file, int line,
+                      mbedtls_ssl_tls13_ticket_flags flag )
+{
+    char str[DEBUG_BUF_SIZE];
+
+    if( NULL == ssl              ||
+        NULL == ssl->conf        ||
+        NULL == ssl->conf->f_dbg ||
+        level > debug_threshold )
+    {
```

Comment:
```suggestion
```

Reply:
This block is copied from another function in this file. And I think the curly brackets is required according to [**Braces placement and block declaration**](https://mbed-tls.readthedocs.io/en/latest/kb/development/mbedtls-coding-standards/#braces-placement-and-block-declaration)